README.md

```diff
@@ -2,39 +2,42 @@
 
 A mobile optimized sliding navigation plugin.
 
-## Dependencies
+[![NPM](https://nodei.co/npm/navitron.png?downloads=true)](https://nodei.co/npm/navitron/)
 
-* [Zepto](http://zeptojs.com/)
-* [Mobify's fork of Velocity.js](http://github.com/mobify/velocity)
-* [Plugin](http://github.com/mobify/plugin)
+## Demo
 
+You can find a simple demo inside the `examples` folder in the repo. Run `grunt examples` to see it in Chrome (mobile device emulation is recommended).
 
-### Velocity
+## Dependencies
```

Comment:
This section just follows our conventions to have this information in the README for quick reference. How verbose we are here is still (IMO) up for debate. I simply followed the format of other [plugins](https://github.com/mobify/pinny/blob/develop/README.md#dependencies).